libs/types/package.json

```diff
@@ -44,6 +44,7 @@
   },
   "devDependencies": {
     "@types/jest": "^27.0.3",
+    "@types/kiosk-browser": "workspace:*",
```

Comment:
Let me know if there is a problem with `@types/kiosk-browser` being a dependency for `libs/types`. I think it makes a little more sense for our hardware types to live in `libs/types` too, not `libs/utils`.

Reply:
I don't see an issue with this. @arsalansufi @eventualbuddha do you?

Reply:
I also don't see any issue with this!

> I think it makes a little more sense for our hardware types to live in libs/types too, not libs/utils.

Also agree with this :100: